pkg/client/apiutil/restmapper.go

```diff
@@ -162,25 +167,34 @@ func (m *mapper) addKnownGroupAndReload(groupName string, versions ...string) er
 	// If no specific versions are set by user, we will scan all available ones for the API group.
 	// This operation requires 2 requests: /api and /apis, but only once. For all subsequent calls
 	// this data will be taken from cache.
-	if len(versions) == 0 {
-		apiGroup, err := m.findAPIGroupByName(groupName)
+	//
+	// Calling this also tells us if the server supports aggregated discovery, which allows
```

Comment:
really small nit. "also tells us if the server supports aggregated discovery"

I guess calling "findAPIGroupByNameAndMaybeAggregatedDiscovery" only tells us that aggregated discovery worked this time (nothing about future calls)

WDYT about:
```go
	// We always run this once, because if the server supports aggregated discovery, this will
	// load everything with two api calls which we assume is overall cheaper.
```

(but feel free to ignore if you prefer the current version or an alternative)

Reply:
Updated